doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -642,6 +646,14 @@ Matter samples
 
     * The documentation with instructions on how to build the sample on the nRF54L15 DK with support for Matter OTA DFU and DFU over Bluetooth SMP, and using internal RRAM only.
     * Link Time Optimization (LTO) to be enabled by default for the ``release`` configuration and ``nrf7002dk/nrf5340/cpuapp`` build target.
+    * All Matter samples that runs Wi-Fi stack to store the Wi-Fi firmware patch in external flash.
```

Comment:
```suggestion
    * Updated all Matter samples that run Wi-Fi stack to store the Wi-Fi firmware patch in external flash.
```
This entry is still in a wrong place (under Matter template sample). I think it needs to be after the Added the Matter manufacturer-specific sample bullet, starting like I suggest above..

Reply:
ahhh, ok right